README.md

```diff
@@ -12,18 +12,9 @@ Run through those steps **including the addition of `/etc/hosts` aliases and the
 
 ### Configure mitxonline and Open edX
 
-**Open edX Tutor** can be used with MITx Online instead of the traditional devstack release. See [Local Open edX Tutor and MITx Online Deployment](docs/source/configuration/tutor.rst) for details.
-
-See [MITx Online Quick Start](docs/source/configuration/quickstart.rst) and [Configure Open edX](docs/source/configuration/open_edx.rst)
-
-### Configuring the CMS
```

Comment:
Why is this being removed? The configure_instance command doesn't configure the CMS and that still needs to be done.

Reply:
The thing GH decided to feature is misleading - I'm referring to the block starting at line 19 (Configuring the CMS). The stuff above it is fine to remove.

Reply:
we are configuring wagtail here https://github.com/mitodl/mitxonline/blob/main/main/management/commands/configure_instance.py#L201

Reply:
:+1: missed that looking through the management command code so this is fine